lib/active_model/serializer.rb

```diff
@@ -98,10 +98,6 @@ def self.adapter
       ActiveModel::Serializer::Adapter.lookup(config.adapter)
     end
 
-    def self.root_name
-      name.demodulize.underscore.sub(/_serializer$/, '') if name
-    end
-
```

Comment:
plz add a changelog entry that this is a breaking change (would be good to see where it was last used?), and I'll merge it

good catch